.github/workflows/main.yml

```diff
@@ -48,6 +48,7 @@ jobs:
         run: |
           cd tests
           ./ci-$TARGET$SUBTARGET
+          cd ..
```

Comment:
BTW, there's also a `working-directory` keyword in GitHub Actions (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun):

> Using the `working-directory` keyword, you can specify the working directory of where to run the command.
> 
> ```ksy
> - name: Clean temp directory
>   run: rm -rf *
>   working-directory: ./temp
> ```

Reply:
Fixed, thanks!